### DISCUSSION TITLE
Dump / gather all transloco keys from angular ?

### DISCUSSION THREAD

**Discussion by jeffschulzusc** [ngneat/transloco#804] [Q&A] [answered]
currently i have in my config:
logMissingKey: isDevMode()

then i look at console warnings for keys that need translation ... 
is there a way to dump all 'static' text that is being piped thru transloco or in calls to transloco.translate ?

thanks

**Comment by shaharkazaz** [comment, marked as answer]
I think that what you are looking for is the [Tranlsoco Keys Manager](https://github.com/jsverse/transloco-keys-manager)

**Comment by jeffschulzusc** [comment]
ah great, thanks much !
